### DISCUSSION TITLE
Fill issues with URL param

### DISCUSSION THREAD

**Discussion by Poolitzer** [community/community#50310] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Hi.

I want to basically make a html button which opens GitHub new issue page, with the title/body of the issue pre filled by whatever I put in the URL as params.

Is that a thing?

**Comment by Majid-J** [comment]
Hey!

I don't think that is possible. However, you could do it this way:
```
// Octokit.js
// https://github.com/octokit/core.js#readme
const octokit = new Octokit({
  auth: 'YOUR-TOKEN'
})

await octokit.request('POST /repos/{owner}/{repo}/issues', {
  owner: 'OWNER',
  repo: 'REPO',
  title: 'Found a bug',
  body: 'I\'m having a problem with this.',
  assignees: [
    'octocat'
  ],
  milestone: 1,
  labels: [
    'bug'
  ],
  headers: {
    'X-GitHub-Api-Version': '2022-11-28'
  }
})
```

Hope this helps 🙂

**Comment by Poolitzer** [comment, marked as answer]
Ha, I actually figured out it is, github uses it for the pre filled templates.

https://github.com/REPO/issues/new?title=Test&body=Abody
